src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalc, Draw {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public double area() {
```

Comment:
The method 'area' should not be implemented here if this class is meant to be a base class for different figures. Each figure should have its own implementation of 'area'. Consider making this method abstract if you want to enforce its implementation in subclasses.